README.md

```diff
@@ -52,23 +59,23 @@ The wasms should be located under `target/release/wbuild/litentry-parachain-runt
 
 Similarly, use `make build-runtime-litmus` and `make build-runtime-rococo` to build the litmus-parachain-runtime and rococo-parachain-runtime, respectively.
 
-## launch a standalone node
+## Launch a standalone node
```

Comment:
I think we can remove this section - it's not widely used anymore :( (and not as important if it doesn't work with the sidechain)